### ISSUE TITLE
Cmd4 shows warnings if the bash scripts are not executables

### ISSUE THREAD

**Issue by jsiegenthaler** [ztalbot2000/homebridge-cmd4#58]
Cmd4 shows warnings if the bash scripts are not executables

As I understand it, the bash script files do not need to be executable. They are .sh files run when the bash command runs them. But the Cmd4 plugin shows a warning in Homebridge, which is unnecessary.

Can you consider updating the plugin to not show warnings for normal .sh files?

**Comment by ztalbot2000** [comment]
Could you please copy/paste the warning you are writing about.

Thx,
John Talbot

**Comment by ztalbot2000** [comment]
Your config.json as well please.

**Comment by jsiegenthaler** [comment]
Here you go:

During platform initialization

```
[6/15/2020, 20:59:24] [Cmd4] Initializing Cmd4 platform...
[6/15/2020, 20:59:24] [Cmd4] Fetching config.json devices.

[6/15/2020, 20:59:24] [Cmd4] Processing accessory: Desktop PC
[6/15/2020, 20:59:24] [Cmd4] The command ./Cmd4Scripts/networkdevice.sh does not exist, It is highly likely the state_cmd will fail. Hint: Do not use wildcards that would normally be expanded by a shell.

[6/15/2020, 20:59:24] Initializing platform accessory 'Desktop PC'...
```

The relevant part of the config.json:

```
        {
            "platform": "Cmd4",
            "name": "Cmd4",
            "outputConstants": false,
            "accessories": [
               {
                    "type": "Switch",
                    "displayName": "Desktop PC",
                    "on": "FALSE",
                    "name": "Desktop PC",
                    "manufacturer": "Gigabyte",
                    "model": "Vista Home Premium",
                    "serialNumber": "abc123",
                    "polling": [
                        {
                            "on": "FALSE",
                            "interval": 2,
                            "timeout": 10000
                        }
                    ],
                    "state_cmd_prefix": "bash",
                    "state_cmd": "./Cmd4Scripts/networkdevice.sh",
                    "state_cmd_suffix": ""
                }
             ]
        }
```

**Comment by ztalbot2000** [comment]
This is interesting and you are correct.  In 'C' it is normal that when you call exec that the file to run be executable.  That is not the case in node.js.  I have the fix and will push it shortly.

**Comment by ztalbot2000** [comment]
fixed with commit 53b8f70.  it will be picked up in next release.
